py/client/pydeephaven/updateby.py

```diff
@@ -0,0 +1,239 @@
+#
+#     Copyright (c) 2016-2023 Deephaven Data Labs and Patent Pending
+#
+from abc import ABC, abstractmethod
+from enum import Enum
+from typing import List, Union
+from .dherror import DHError
+from pydeephaven.proto import table_pb2
+
+
+GrpcUpdateByOperation = table_pb2.UpdateByRequest.UpdateByOperation
+GrpcUpdateByColumn = GrpcUpdateByOperation.UpdateByColumn
+GrpcUpdateBySpec = GrpcUpdateByColumn.UpdateBySpec
+GrpcUpdateByEma = GrpcUpdateBySpec.UpdateByEma
+GrpcUpdateByEmaOptions = GrpcUpdateByEma.UpdateByEmaOptions
+GrpcUpdateByEmaTimescale = GrpcUpdateByEma.UpdateByEmaTimescale
+GrpcUpdateByEmaTicks = GrpcUpdateByEmaTimescale.UpdateByEmaTicks
+GrpcUpdateByEmaTime = GrpcUpdateByEmaTimescale.UpdateByEmaTime
+GrpcMathContext = table_pb2.MathContext
```

Comment:
1) Do these need to be public?
2) If they do need to be public, they should be documented.